Source/iOS/OIDAuthorizationUICoordinatorIOS.m

```diff
@@ -25,6 +25,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** @var gSafariViewControllerFactory
+    @brief The global/shared Safari view controller factory. Responsible for creating all new
+        instances of @c SFSafariViewController.
+ */
+static id<OIDSafariViewControllerFactory> __nullable gSafariViewControllerFactory;
```

Comment:
Another option would be to store this as an instance variable. This would be better for testability.